sklearn/metrics/_classification.py

```diff
@@ -181,6 +187,12 @@ def accuracy_score(y_true, y_pred, *, normalize=True, sample_weight=None):
     sample_weight : array-like of shape (n_samples,), default=None
         Sample weights.
 
+    zero_division : {"warn", 0.0, 1.0, np.nan}, default="warn"
+        Sets the value to return when there is a zero division.
+
+        Notes:
+        - If set to "warn", this acts like 0, but a warning is also raised.
```

Comment:
To my ears "acts like 0" is not very clear. A suggestion:

```suggestion
        - If set to "warn", this behaves like a 0.0 input, but a warning is also 
        raised.
```

Reply:
Done